source/portable/NetworkInterface/DriverSAM/gmac_SAM.c

```diff
@@ -621,6 +621,9 @@ uint32_t gmac_dev_read( gmac_device_t * p_gmac_dev,
         return GMAC_RX_NO_DATA;
     }
 
+    /* Return the number of bytes received. */
+    *p_rcv_size = bytesLeft;
```

Comment:
Is it necessary to compare with frame size?
The `*p_rcv_size` might be different when `ul_frame_size` is smaller than `bytesLeft + 2`.
```
if( bytesLeft + 2 <= ul_frame_size )
{
    *p_rcv_size = bytesLeft;
}
else
{
    *p_rcv_size = ul_frame_size;
}
```